src/ProgramBlockEditor.test.js

```diff
@@ -180,6 +191,105 @@ test('When a step is clicked, action panel should render next to the step', () =
     }
 });
 
+
+describe("Expand button and add nodes", () => {
+    test("The expand button should work when there are no program blocks.", () => {
+        expect.assertions(2);
+    
+        const { wrapper } = createMountProgramBlockEditor({
+            program: []
+        });
+    
+        expect(getEnabledAddNodeCount(wrapper)).toBe(1);
+    
+        const expandNode = getAddNodeExpandButton(wrapper);
```

Comment:
Rename to: `toggleSwitch` or `expandAddNodeToggleSwitch`

Reply:
Done, throughout.